stats/kafka/config.go

```diff
@@ -20,6 +20,7 @@
 
 package kafka
 
+import "C"
```

Comment:
I think you've mistakenly [imported cgo](https://golang.org/cmd/cgo/) ... pretty sure it was because of some goimports integration or IDE :D

Reply:
Thank you for warning me. I really not focus on this import. Let's me check on it.